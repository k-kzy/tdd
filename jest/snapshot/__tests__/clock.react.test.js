'use strict';

import React from 'react';
import Clock from '../Clock.react';
import renderer from 'react-test-renderer';

jest.useFakeTimers();
Date.now = jest.fn(() => 11482363367071);

it('renders correctly', () => {
  const tree = renderer.create(<Clock />).toJSON();
  expect(tree).toMatchSnapshot();
});
