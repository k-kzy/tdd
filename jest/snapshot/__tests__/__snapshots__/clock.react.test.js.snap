// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p>
   seconds have ellapsed since the UNIX epoch.
</p>
`;
